appserver/admingui/common/src/main/java/org/glassfish/admingui/common/handlers/SecurityHandler.java

```diff
@@ -277,7 +278,7 @@ public static void saveRealm(HandlerContext handlerCtx) {
                 continue;
             }
             sb.append(oneProp.get("name")).append("=");
-            String value = ((String) oneProp.get("value")).replaceAll("\\\\", "\\\\\\\\");
+            String value = ((String) oneProp.get("value"));
```

Comment:
the outer brackets of the value are no longer needed :)

Reply:
Thank you for noticing that, I fixed.